composer.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "laravel/laravel.com",
```

Comment:
This probably should be updated to `mage-os/devdocs` or something like this?  
The same for the description :)

Reply:
Yeh, good catch